### DISCUSSION TITLE
80M image-formula pairs dataset

### DISCUSSION THREAD

**Discussion by lixumin-zai** [OleehyO/TexTeller#66] [General]
I'm working on training and testing Florence-2 and would like to use your `80M` image-formula pairs dataset. Could you please share it with me?

**Comment by OleehyO** [comment]
The dataset is expected to be open-sourced after August. Please wait a moment.

**Reply by lixumin-zai** [reply]
https://github.com/lixumin-zai/Florence-2-finetune
If you have the time and resources, you can use this code to train.

**Reply by lixumin-zai** [reply]
I see the dataset has been published on huggingface, but I don't have permission.
I want to try using it to train Florence-2  model.

**Reply by weichong1994** [reply]
Hi @OleehyO, Thanks for this amazing work, very inspiring! It is possible to approve my access to the dataset on huggingface? I would like to try it out on qwenvl. Thanks for the help!
